doc/dox_comments/header_files/ssl.h

```diff
@@ -2698,6 +2698,72 @@ WOLFSSL_API void wolfSSL_load_error_strings(void);
 */
 WOLFSSL_API int  wolfSSL_library_init(void);
 
+/*!
+    \brief This function sets the Device Id.
```

Comment:
Let's also add that this sets the device ID at the WOLFSSL session level.  Also, I don't see a "\param" for the devId parameter?